migrations/174_workspaces__create.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TYPE enum_workspace_state AS ENUM ('launching', 'running', 'stopped', 'uninitialized');
```

Comment:
The order of these doesn't really matter, but I'd be inclined to order them as we expect they will be used (that is, `uninitialized`, `stopped`, `launching`, `running`).